### PR TITLE
Adding tioanime.com Presence

### PR DESCRIPTION
**Pull request by u0f**
imagen
imagen

Paused chapter
imagen
Playing chapter
imagen

### CODE REVIEW

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Comment by u0f** [comment]
> 
> 
> Add more features to the presence (e.g. settings, multiLang, buttons, etc)

Add for adding? You don't really need to put any of that in.
Mutlilang for what? The website is only in spanish.
Settings for what? I don't need it, is a really normal presence 
Buttons? Idk i'll add some

File: websites/T/Tio Anime/presence.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+   * @param {Number} videoDuration Video duration seconds
+   */
+function getTimestamps(
+  videoTime: number,
+  videoDuration: number
+): Array<number> {
+  const startTime = Date.now(),
+    endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+  return [Math.floor(startTime / 1000), endTime];
+}
```
Comment (Slowlife01) [inline websites/T/Tio Anime/presence.ts:23]: Again

File: websites/T/Tio Anime/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "logo": "https://i.imgur.com/KJUHptb.png",
+  "thumbnail": "https://i.imgur.com/vNi2bMS.png",
+  "version": "1.0.0",
+  "color": "#18142f",
+  "tags": [
+    "anime"
+  ],
+  "url": "tioanime.com",
+  "description": {
+    "en": "Watch any anime in spanish!",
```
Comment (Slowlife01) [inline websites/T/Tio Anime/dist/metadata.json:18]: *And you changed it back.*

**Review by Slowlife01 — CHANGES_REQUESTED** [review]

**Comment by Slowlife01** [comment]
Also update your fork

**Comment by u0f** [comment]
idk why schema validation check is failing

File: websites/T/Tio Anime/presence.ts [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  duration: 0,
+  currentTime: 0,
+  paused: true
+};
+
+function getTimestamps(videoTime: number, videoDuration: number) {
+  const startTime = Date.now(),
+    endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+  return [Math.floor(startTime / 1000), endTime];
+}
```
Comment (Slowlife01) [inline websites/T/Tio Anime/presence.ts:16]: No. Use `Presence.getTimestanps()`

Comment (u0f) [reply websites/T/Tio Anime/presence.ts:16]: OOH, okay tysm ^^

File: websites/T/Tio Anime/presence.ts [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  duration: 0,
+  currentTime: 0,
+  paused: true
+};
+
+/*function getTimestamps(videoTime: number, videoDuration: number) {
+  const startTime = Date.now(),
+    endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+  return [Math.floor(startTime / 1000), endTime];
+}*/
```
Comment (Slowlife01) [inline websites/T/Tio Anime/presence.ts:16]: Remove it

Comment (u0f) [reply websites/T/Tio Anime/presence.ts]: My bad

**Review by Slowlife01 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**